src/marker/custom.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 FastLabs Developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt::Debug;
+use std::fmt::Formatter;
+
+use crate::Marker;
+
+// TODO(tisonkun): use trait alias when it's stable - https://github.com/rust-lang/rust/issues/41517
```

Comment:
Perhaps not my ID here XD

Reply:
Just to align with the same comment in CustomLayout